src/NServiceBus.Core.Tests/AssemblyScanner/When_scanning_directory_with_nested_directories.cs

```diff
@@ -12,6 +13,7 @@ public class When_scanning_directory_with_nested_directories
         public void Should_not_scan_nested_directories_by_default()
         {
             var busConfiguration = new BusConfiguration();
+            busConfiguration.ExcludeTypes(typeof(When_using_initialization_with_non_default_ctor.FeatureWithInitialization));
```

Comment:
This kind of sucks, should this be a acpt test instead? (can be a different pull though) cc @SimonCropp